supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -1,10 +1,59 @@
 [[release-notes]]
 = Release notes
 
-[[release-notes-doc-texts]]
-== Release notes doc texts
+A release note explains specific product behavior, capability, or problem relevant to a product version. Release notes for a particular product version are collected in a document that is published when the new version is released.
 
-A _doc text_ is a short description of an engineering Bugzilla or Jira issue that is published in product Errata advisories and release notes. Engineering typically supplies draft content, which is a summary of the issue, then technical writers edit or rewrite the draft content in accordance with the _IBM Style_ guide. You can write doc texts in more than one way, depending on the issue type:
+[[style-advice-for-release-note-texts]]
+== Style advice for release note texts
```

Comment:
I'm not a huge fan of the name of this section, or the fact that it mixes the actual paragraph text and the heading, but then heading has its own section down below.

Perhaps consider making (and naming) this section _just_ about the actual paragraph text, and leave any guidance for headings in the heading section?

You could also consider whether there is anything else that could be pulled out into their own subsections, like you did with headings and tenses. There is just a lot of bullets here, maybe they could be grouped further. Perhaps, capitalization, for example.

I personally would rather see shorter categories/sections, even if there were only 2-3 bullets, rather than longer lists of bullets.